index.js

```diff
@@ -59,20 +62,24 @@ var run = function() {
   log.info('Beginning couch2pg and xmlforms run at ' + new Date());
 
   var runErrored = false;
-
   return importer.importAll()
   .catch(function(err) {
     log.error('Couch2PG import failed');
     log.error(err);
 
+    // CouchDB will return a json response with a 401 status code in case of
+    // authentication failure
+    if ((typeof err == "object" && err.status == 401)){
```

Comment:
You don't need multiple parens here, you don't need to check the type and you do want to use triple equals instead of double equals. I should probably add a jshint step to the travis build so it catches stuff like that.

Anyway, one suggested replacement would be: `if (err && err.status === 401)`.

You should also drop the comment, this code self-documents (a 401 is a known status code meaning unauthorised and the block sets a var called `authError`).